components/service/pocket/src/main/java/mozilla/components/service/pocket/PocketStory.kt

```diff
@@ -0,0 +1,71 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.pocket
+
+/**
+ * A Pocket story downloaded from the Internet and intended to be displayed in the application.
+ */
+sealed class PocketStory {
+    /**
+     * Title of the story.
+     */
+    abstract val title: String
+
+    /**
+     * Url where the story can be full read.
+     */
+    abstract val url: String
+
+    /**
+     * A Pocket recommended story.
+     *
+     * @property title the title of the story.
+     * @property url a "pocket.co" shortlink for the original story's page.
+     * @property imageUrl a url to a still image representing the story.
+     * @property publisher optional publisher name/domain, e.g. "The New Yorker" / "nationalgeographic.co.uk"".
+     * **May be empty**.
+     * @property category topic of interest under which similar stories are grouped.
+     * @property timeToRead inferred time needed to read the entire story. **May be -1**.
+     */
+    data class PocketRecommendedStory(
```

Comment:
I think we can drop the `Pocket` prefix since all of these classes are already within `PocketStory`, but up to you.